week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,18 @@
 // Add comments to explain what this function does. You're meant to use Google!
-function getNumber() {
-    return Math.random() * 10;
+
+function getNumber() { 
+    return Math.random() * 10;// this funtion returns a random number then multiplys by 10.
```

Comment:
`math.random()` generates only random numbers between 0 and 1. so if we multiply by 10 what we get ??